app/models/note.rb

```diff
@@ -0,0 +1,8 @@
+class Note < ActiveRecord::Base
+  attr_accessible :content, :title
+
+  belongs_to :notable, polymorphic: true
```

Comment:
when doing a `polymorphic` association, I usually find it a good idea to also do a inclusion validation on `notable_type`. That way when you look at the Note model you can see everything that could be a notable.

```suggestion
  belongs_to :notable, polymorphic: true
  validates :notable_type, inclusion: [Student].map(&:to_s)
```